frontend/public/components/stateful-set.jsx

```diff
@@ -63,7 +63,7 @@ const Row = ({obj: ss}) => <div className="row co-resource-list__item">
 const Details = ({obj: ss}) => <React.Fragment>
   <div className="co-m-pane__body">
     <Heading text="StatefulSet Overview" />
-    <ResourceSummary resource={ss} podSelector="spec.podSelector" showNodeSelector={false} />
+    <ResourceSummary resource={ss} podSelector="spec.selector" showNodeSelector={false} />
```

Comment:
Looks like spec.selector is used in ResourceSummary by default

Reply:
thanks